letsencrypt/acme/other.py

```diff
@@ -0,0 +1,86 @@
+"""JSON objects in ACME protocol other than messages."""
```

Comment:
I honestly can't think of a better name. Any suggestions? FYI in my recent commits `other.py` contains both `Signature` and `JWK`.